docs/doc_5_best_practices.md

```diff
@@ -69,7 +69,7 @@ The golang `ìnterface{}` type allows you to use the following types as input ar
 
 - Basic types (integer, boolean)
 - Your own structures
-- map[string]interface{} for free-form objects
+- map[string]any for free-form objects
```

Comment:
interface{} -> any leftover

Reply:
think free form are just any right?

Reply:
yes, although here I'm not changing doc, just interface{} to alias any

Reply:
`any for free-form objects` is the correct form. Do you mind applying it?

Reply:
@wtrocki sorry I'm not sure I'm following you, can you give me an example where I should apply it?

Reply:
```suggestion
- any for free-form objects
```